csharp.benchmark/FloatTimeSeriesBase.cs

```diff
@@ -7,33 +7,21 @@ public abstract class FloatTimeSeriesBase
     {
         protected static Column[] CreateFloatColumns()
         {
-            return new Column[]
-            {
-                new Column<DateTime>("DateTime", LogicalType.Timestamp(true, TimeUnit.Millis)),
-                new Column<int>("ObjectId"),
-                new Column<float>("Value")
-            };
+            return new Column[] {new Column<DateTime>("DateTime", LogicalType.Timestamp(true, TimeUnit.Millis)), new Column<int>("ObjectId"), new Column<float>("Value")};
         }
 
         protected static (DateTime[] dates, int[] objectIds, float[][] values, int numRows) CreateFloatDataFrame(int numDates)
         {
             var rand = new Random(123);
 
-            var dates = Enumerable.Range(0, numDates)
-                .Select(i => new DateTime(2001, 01, 01) + TimeSpan.FromHours(i))
-                .Where(d => d.DayOfWeek != DayOfWeek.Saturday && d.DayOfWeek != DayOfWeek.Sunday)
-                .ToArray();
+            var dates = Enumerable.Range(0, numDates).Select(i => new DateTime(2001, 01, 01) + TimeSpan.FromHours(i)).Where(d => d.DayOfWeek != DayOfWeek.Saturday && d.DayOfWeek != DayOfWeek.Sunday).ToArray();
```

Comment:
Same remark as above. The previous style was more readable (one chained method call per line).